### PR TITLE
[BT-668] Add deploy instructions

### PR DESCRIPTION
**Pull request by breilly2**
https://broadworkbench.atlassian.net/browse/BT-668

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,7 +41,32 @@ Note that there is currently no separate development environment for mixpanel, s
...
     yarn run lint
     ```
-   
+
+## Deploying
+**Note**: The deploy process will overwrite any `./config.json` you may currently be using. Before running these scripts, make a backup of this file if you have any interesting config that you'd like to preserve (i.e. not just unmodified copies of files in `./config/`).
+
+Bard deployment is fully manual. You must have the repo cloned locally. There are 2 scripts in `./scripts/`:
+* `deploy-all-but-prod.sh`: Deploys to dev, alpha, perf, and staging using their respective config files in `./config/`. Run this with your @broadinstitute.org account.
+* `deploy-prod.sh`: Deploys to prod using `./config/prod.json`. Run this with your @test.firecloud.org account.
```
Comment (jgainerdewar) [inline README.md]: Why not our @firecloud.org accounts?

**Review by jgainerdewar — APPROVED** [review]
Thank you! Had one question, not a blocker for merging.

**Review by kraefrei — APPROVED** [review]
LGTM, although also curious about Janet's question.

File: README.md (hunk 1 above)
Comment (breilly2) [reply README.md]: You're right. That's a mistake on my part. Thanks for catching it!

**Pull request merged**